### DISCUSSION TITLE
快捷菜单

### DISCUSSION THREAD

**Discussion by gbsix** [shuax/MouseInc.Settings#296] [Ideas]
快捷菜单目前只能单个条目编辑，操作起来略显繁琐，可否改变一下样式，批量编辑然后整体保存，类似下面这样：

![2](https://user-images.githubusercontent.com/98303833/174466619-9a4cc781-a117-472b-907c-a09b06bf0cb6.jpg)
![1](https://user-images.githubusercontent.com/98303833/174466621-ddea2256-7f9b-4083-8433-4dbfe027b86a.jpg)

**Comment by Ctrl-Ling** [comment]
可以直接修改json文件

**Reply by gbsix** [reply]
毕竟修改json乱七八糟的不太友好

**Reply by shuax** [reply]
> 毕竟修改json乱七八糟的不太友好

胆大心细，不要怕
